first-party/build.gradle

```diff
@@ -292,6 +294,17 @@ subprojects {
   /* TEST COVERAGE */
 }
 
+/* SONARQUBE */
+apply plugin: "org.sonarqube"
```

Comment:
Oh, does this need to be applied at the root project?

Can you confirm that applying a plugin this way doesn't apply it in sub-projects (if that is what is intended)? (The reason we have `apply false` for other plugins and apply them only in the `subprojects {}` block is to prevent the root project to load it while applying it in subprojects.) I think it is still the case that applying a plugin at root makes it applied in subprojects.

Reply:
I see. For some reason, the properties defined in the sonarqube task  (such as `sonar.host.url`) weren't getting set when I had the task inside `subprojects {}` so I decided to move the task to the project root. Let me try this out again. There are a couple of different ways to set properties for sonarcloud (using gradle.properties is one) so I'll check if it is possible to avoid having sonarqube at the root project.

Reply:
Just verified this again and there is another issue with placing sonarqube in `subprojects{}`. When sonarqube task is placed in `subprojects{}` instead of the root project, the metrics for each subproject gets overwritten by the next subproject on sonarcloud.io. According to this [sample](https://github.com/SonarSource/sonar-scanning-examples/tree/master/sonarqube-scanner-gradle/gradle-multimodule-coverage) and [these instructions](https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-gradle/), it seems like we need to have the task at the root project.

Reply:
I see. According to the instructions, looks like it is correct to run the plugin in all projects. Although loading a plugin in the root project may affect dependency conflicts, I think in this case it is correct to apply it at the root. That is, don't set `apply false` (make a comment about it like in `org.jetbrains.gradle.plugin.idea-ext`); then no need for `apply plugin: 'org.sonarqube'`, I guess.
```gradle
  // apply so we can correctly configure the test runner to be gradle at the project level
  id 'org.jetbrains.gradle.plugin.idea-ext' version '0.10'
  // apply ...
  id 'org.sonarqube' version '3.3' apply false
```

Nit: Use single quotes instead of double for constants.

Reply:
Sounds good. 

> dependency conflicts,
Oh that makes sense. I see why we want to be cautious about applying the plugin at the root project level. 

> Nit: Use single quotes instead of double for constants.
Thank you! Ah I always forgot to do this.